### PR TITLE
Update marketo api

### LINKED ISSUE #129: Deprecated marketo endpoint hasn't been updated
The deprecated marketo endpoint hasn't been updated on charmed-osm.com

image

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done
- Post forms to https://ubuntu.com/marketo/submit
- Decapitalise input names
- Update hidden inputs
- Remove recaptcha

## QA

1. Go to  https://charmed-osm-com-131.demos.haus/#get-in-touch or https://charmed-osm-com-131.demos.haus/contact-us
2. Fill in the form and submit it
3. Check the form is submitted and redirects to a thank you page on charmed-osm.com
4. Check the submitted data is on the list in marketo (form id : 3471)

## Issue / Card

Fixes #129

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -291,20 +291,9 @@ <h3>Sign up for our monthly Cloud newsletter</h3>
...
-            <input type="hidden" name="lpurl" value="https://pages.canonical.com/Insights-Subscription_Insights-Subscription-test.html?cr={creative}&amp;kw={keyword}">
-            <input value="1212" class="mktoField mktoFieldDescriptor" name="formid" type="hidden">
-            <input type="hidden" name="ret" value="http://charmed-osm.com/thank-you">
-            <input value="066-EOV-335" class="mktoField mktoFieldDescriptor" name="munchkinId" type="hidden">
-            <input type="hidden" name="kw" value="">
-            <input type="hidden" name="cr" value="">
-            <input type="hidden" name="searchstr" value="">
-            <input type="hidden" name="_mkt_disp" value="return">
-            <input type="hidden" name="_mkt_trk" value="">
+            <input value="1212" p-buttonname="formid" type="hidden">
```
Comment (anthonydillon) [inline templates/index.html:296]: Extra p-button here

File: templates/index.html [hunk 2]
```diff
@@ -291,20 +291,9 @@ <h3>Sign up for our monthly Cloud newsletter</h3>
...
+            <span><button type="submit" class="p-button" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'Cloud newsletter', 'eventLabel' : 'Sign-up for news updates', 'eventValue' : undefined });">Subscribe now</button></span>
             <input type="hidden" name="Consent_to_Processing__c" value="yes">
-            <input value="1959" class="mktoField mktoFieldDescriptor" name="lpId" type="hidden">
-            <input value="30" class="mktoField mktoFieldDescriptor" name="subId" type="hidden">
-            <input type="hidden" name="lpurl" value="https://pages.canonical.com/Insights-Subscription_Insights-Subscription-test.html?cr={creative}&amp;kw={keyword}">
-            <input value="1212" class="mktoField mktoFieldDescriptor" name="formid" type="hidden">
-            <input type="hidden" name="ret" value="http://charmed-osm.com/thank-you">
```
Comment (anthonydillon) [inline templates/index.html:301]: Send this form doesn't have a return URL

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some suggestions in the code

**Comment by minkyngkm** [comment]
Thanks @anthonydillon for reviewing.

QA done for the `/contact-us` page
image

One question is [the interactive form](https://charmed-osm-com-131.demos.haus/#get-in-touch)'s form id is same as this static [contact-us form](https://charmed-osm-com-131.demos.haus/contact-us) even though the required input fields are different. I failed QAing the interactive form. 😔

**Comment by anthonydillon** [comment]
The form id is ok to be the same but the interactive form textarea update is not working: 
image

File: templates/contact-us/index.html [hunk 3]
```diff
@@ -14,36 +14,36 @@ <h1>Contact Canonical</h1>
...
-              <input required id="Email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired" />
+            <li class="p-list__item">
+              <label for="email">Email address:</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"/>
             </li>
-            <li class="mktFormReq mktField p-list__item">
-              <label for="Phone" class="mktoLabel">Phone number:</label>
-              <input required id="Phone" name="Phone" maxlength="255" type="tel" class="mktoField mktoTelField mktoRequired" />
+            <li class="p-list__item">
+              <label for="phone">phone number:</label>
```
Comment (anthonydillon) [inline templates/contact-us/index.html:33]:
```suggestion
              <label for="phone">Phone number:</label>
```

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: with a small content update

**Pull request merged**